README.md

```diff
@@ -2,9 +2,54 @@
 
 [![Circle CI](https://circleci.com/gh/SumOfUs/Champaign/tree/master.svg?style=shield)](https://circleci.com/gh/SumOfUs/Champaign/tree/master) [![Coverage Status](https://coveralls.io/repos/github/SumOfUs/Champaign/badge.svg?branch=master)](https://coveralls.io/github/SumOfUs/Champaign?branch=master) [![Code Climate](https://codeclimate.com/github/SumOfUs/Champaign/badges/gpa.svg)](https://codeclimate.com/github/SumOfUs/Champaign)
 
-Champaign is a digital campaigning platform built by SumOfUs. It is now production ready and we are in the process of migrating all our campaigning operations from ActionSweet to Champaign. If you're interested in collaborating on the project with us, or have ideas or recommendations, please get in touch!
+Champaign is an open source digital campaigning platform built by SumOfUs. It provides powerful tools for customizing and experimenting with campaign pages, while streamlining the process of setting them up.
 
-## Installation
+If you're interested in collaborating on the project with us, or have ideas or recommendations, please get in touch!
+
+## Development setup
+
+* Install gem dependencies by running `gem install bundler` and then `bundle install`.
+* Install node dependencies by running `npm install`
+* Setup your db connection by running `cp config/env.template.yml config/env.yml` and edit
+  `config/env.yml` with your development database information.
+* Create the development databases: `bundle exec rake db:create`
+* Run migrations: `bundle exec rake db:migrate`
```

Comment:
It's recommended to use `rake db:schema:load` rather than migrations the first time you get an existing rails project up and running.